dqn.cpp

```diff
@@ -22,29 +22,29 @@ void HasBlobSize(caffe::Net<Dtype>& net,
 }
 
 void DQN::LoadTrainedModel(const std::string& model_bin) {
-  net_->CopyTrainedLayersFrom(model_bin);
+  net_actor_->CopyTrainedLayersFrom(model_bin);
 }
 
-void DQN::RestoreSolver(const std::string& solver_bin) {
-  solver_->Restore(solver_bin.c_str());
+void DQN::RestoreSolver(const std::string& solver_actor_bin) {
+  solver_actor_->Restore(solver_actor_bin.c_str());
```

Comment:
Same comment as above - either we need to take 2 strings as input or have a separate method for Restoring the Critic's solver.